sqldelight-compiler/src/main/kotlin/com/squareup/sqldelight/model/Table.kt

```diff
@@ -32,6 +33,7 @@ internal class Table(
     nameAllocators: MutableMap<String, NameAllocator>
 ) {
   internal val name = rule.table_name().text
+  internal val javadoc = rule.javadocText()
```

Comment:
inline the method as `javadocText(rule.JAVADOC_COMMENT())` since its pretty small